css-brochure-form/index.html

```diff
@@ -16,7 +16,7 @@ <h1>LEARN MORE</h1>
     <div>
       <form>
         <div>
-          <label for="email-login"><h3>COMPANY EMAIL</h3></label>
+          <label for="email-login"><bold>COMPANY EMAIL</bold></label>
```

Comment:
`bold` is not an element type. I want you to apply `font-weight: bold` to the `label` element in your stylesheet.